website/content/docs/concepts/password-policies.mdx

```diff
@@ -360,6 +360,39 @@ This policy generates 8 character passwords from the charset `abcde01234` and re
 character from `01234` to be in it, but does not require any characters from `abcde`. The password
 `04031945` may result from this policy, even though no alphabetical characters are in it.
 
+## Default Password Policy
+
+Vault ships with a Default Password Policy and this gets applied to any password that Vault 
+generates where a custom password policy is not explicitly set. The default password policy is 
+defined below.
+
+<details>
+<summary>20 characters with at least 1 uppercase, 1 lowercase, 1 number, and 1 dash character.</summary>
+
+```hcl
```

Comment:
I'm confused about what the HCL example is for. Are we expecting readers to copy it and use it somewhere in their config?

Reply:
The HCL extract here is the HCL representation of the default password policy and I added it merely as a reference. Happy to remove it!

Reply:
I added the HCL equivalent of the default password policy merely as a reference for customers wanting to write their own, but i guess a reference already exists [here](https://developer.hashicorp.com/vault/docs/concepts/password-policies#password-policy-syntax). Happy to remove it if it seems redundant.

Reply:
I'm not against keeping it :)